.github/workflows/pull_requests.yaml

```diff
@@ -46,12 +52,17 @@ jobs:
           cache: true
       - name: Run locally Docker build
         run: docker build -t weaviate:${{ github.sha }} .
+      - name: Skip vulnerabilities
+        if: ${{ env.SKIPPED_VULN != '' }}
+        run: |
+          echo "${{ env.SKIPPED_VULN }}" | tr ' ' '\n' > .trivyignore
       - name: Run Trivy vulnerability scanner for the built image
         uses: aquasecurity/trivy-action@0.20.0
         with:
           image-ref: 'weaviate:${{ github.sha }}'
           exit-code: '1'
           format: 'table'
+          ignore-unfixed: true
```

Comment:
does this setting make that it will ignore everything?

Reply:
Usually, the CVEs have different states. You can have a CVE which has been identified, but it hasn't been fixed, and it will appear as `unfixed`. To avoid that CVES of that type block us from merging patches, it was included this flag, so it won't make the job fail for CVEs which do not have a fix released for them yet.